apps/web/pages/documents/[id]/recipients.tsx

```diff
@@ -415,7 +415,7 @@ const RecipientsPage: NextPageWithLayout = (props: any) => {
                       </div>
                     </div>
                   </div>
-                  <div className="mt-5 sm:mt-6 sm:grid sm:grid-flow-row-dense sm:grid-cols-2 sm:gap-3">
+                  <div className="flex justify-end gap-3 mt-5 sm:mt-6 sm:grid sm:grid-flow-row-dense sm:grid-cols-2 sm:flex-none sm:jus">
```

Comment:
sm:jus looks wrong

Reply:
@litaesther10

Reply:
Yeah! I didn’t remove it properly! On it now...